config/locales/layouts/shared/sidebar.en.yml

```
@@ -12,21 +12,23 @@ en:
         calendars: Calendars
         committee_units: Committee Units
         committees: Committees
-        courses: Courses
+        course_enrollments: Course Enrolments
```

```suggestion
        course_enrollments: Course Enrollments
```

(British vs American)